.github/hooks/pre-commit

```diff
@@ -5,6 +5,11 @@
 # ln -s ../../.github/hooks/pre-commit .git/hooks/pre-commit
 #
 
+if [ ! -x "$(command -v circleci)" ]; then
```

Comment:
Wouldn't it make more sense, to only check for this, if the config was changed? Otherwise it is perfectly fine to commit without having circleci locally? Can make onboarding more complicated than it needs to be or do i miss something here?

Reply:
This only fixes the commit hook. If you do not have circleci in your PATH, then the circle ci config is removed (happened to me).

Other changes should be made in another PR. 

This only fixes something and does not change it.

Reply:
Created an issue: https://issues.tuerantuer.org/browse/IGAPP-855